<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1004</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="Canvas" name="stl_viewer" native="true">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>500</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Generate Machine Code</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QTabWidget" name="tabWidget">
              <property name="minimumSize">
               <size>
                <width>325</width>
                <height>100</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab_1">
               <attribute name="title">
                <string>Slice</string>
               </attribute>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="LayerDraw" name="svg_viewer" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>310</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="autoFillBackground">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_2">
               <attribute name="title">
                <string>Raster</string>
               </attribute>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLabel" name="layerNum_label">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Layer #</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QSlider" name="verticalSlider">
                <property name="mouseTracking">
                 <bool>true</bool>
                </property>
                <property name="toolTipDuration">
                 <number>1</number>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="invertedAppearance">
                 <bool>false</bool>
                </property>
                <property name="invertedControls">
                 <bool>false</bool>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksBothSides</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QTabWidget" name="tabWidget_2">
           <property name="tabShape">
            <enum>QTabWidget::Rounded</enum>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="machine_tab">
            <attribute name="title">
             <string>Machine</string>
            </attribute>
            <widget class="QLabel" name="systemImage">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>61</width>
               <height>51</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QWidget" name="">
             <property name="geometry">
              <rect>
               <x>90</x>
               <y>10</y>
               <width>216</width>
               <height>40</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="machineLabel">
                <property name="text">
                 <string>Machine: MSAM Research Platform</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="processLabel">
                <property name="text">
                 <string>Process: Hybrid AM Process</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>Machine Settings</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tab_4">
            <attribute name="title">
             <string>Page</string>
            </attribute>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>canvas.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LayerDraw</class>
   <extends>QWidget</extends>
   <header>layerdraw.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget_2</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
